src/config.ts

```diff
@@ -60,6 +60,11 @@ class Config {
     };
   }
 
+  // Hauki is not production ready; disable all features that require it
+  static get enableHauki() {
+    return false;
+  }
```

Comment:
Your impression is along the lines I intended. I don't entirely understand how I should clear up the confusion here. When the return value of this field is `true`, Hauki should be enabled.

I didn't use an env variable here because the ticket did not call for it and it's trivial to add it later. I'm happy to add one for this setting now.

Reply:
it confuses me too. The name feels it's a void function that sets it from false to true. Something like "isHaukiEnabled" would be better.

Reply:
Ah, now I understand. I thought about this more in terms of a config than code. E.g. a person clicking through an ui, what should the checkbox say; what command you as a user are giving to the application. I'll change this.

Reply:
a very trivial thing anyway :D